src/browser/ui/dom/DOMPropertyOperations.js

```diff
@@ -61,7 +61,7 @@ if (__DEV__) {
     // logging too much when using transferPropsTo.
     warning(
       standardName == null,
-      'Unknown DOM property ' + name + '. Did you mean ' + standardName + '?'
+      'Unknown DOM property ' + name + (value == null ? '': ', value: ' + value) + '. Did you mean ' + standardName + '?'
```

Comment:
Updated the code. Not sure about the markdown comment (if escaping is what you meant - I used escapeTextForBrowser function).